spec/factories/files.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :file, class: Hash do
+    initialize_with {
+      {
+        'id' => '1',
+        'url' => 'http://gap-test.com/file.zip'
```

Comment:
do we expect having `type` returned like we have in other models? if so it probably will be dynamic based on passed `file_type`?